test/framework/resources/aws/services/eks.go

```diff
@@ -23,7 +23,10 @@ import (
 type EKS interface {
 	DescribeCluster(clusterName string) (*eks.DescribeClusterOutput, error)
 	CreateAddon(addon string, clusterName string) (*eks.CreateAddonOutput, error)
+	CreateAddonWithVersion(addon string, clusterName string, addonVersion string) (*eks.CreateAddonOutput, error)
```

Comment:
If only change is the version, then I think we should create an AddonInput struct and pass it to the method, instead of creating a new method only for version. That way it would be easy to extend to other methods.

Reply:
I created a separate func because we have to add an additional parameter ```aws.String("OVERWRITE")```, for ```ResolveConflicts``` in case a version is specified in the eks.go file. If we have one function, we will have to perform a check there and modify the struct to set```ResolveConflicts``` parameter.